docs/source/development/pull_request.md

```diff
@@ -237,6 +237,12 @@ Alternatively, you can run a single specific environment through `tox -e <env>`.
   $ tox -e docs
   ```
 
+- **Additional arguments passed to pytest via tox:** e.g., only run `doctests` in `linalg`.
+
+  ```shell
+  $ tox -e py39 -- src/probnum/linalg
```

Comment:
```suggestion
  $ tox -e py3 -- src/probnum/linalg
```
I removed the specific python version to ensure this runs on most people's machines.